### DISCUSSION TITLE
ABCD data management contact info?

### DISCUSSION THREAD

**Discussion by JaylenSantos** [now-i-know-my-abcd/docs#79] [Where in the World is…? (Help)]
I continue to have issues with KSADS data that I believe stems from an error. Does anybody know if there is an ABCD data management contact?

**Comment by clarefmccann** [comment]
Hi Jaylen! Here is a page with contact info: https://abcdstudy.org/contact/

I think you would want to reach out to Dr. Anders Dale... but if you have specific questions, maybe someone here can help!
